modules/joinmarket.nix

```diff
@@ -0,0 +1,222 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.joinmarket;
+  inherit (config) nix-bitcoin-services;
+  secretsDir = config.nix-bitcoin.secretsDir;
+
+  torAddress = builtins.head (builtins.split ":" config.services.tor.client.socksListenAddress);
+  configFile = builtins.toFile "config" ''
```

Comment:
Where's that config from? Perhaps add comment. Do we need to state all these options, or does joinmarket pick the defaults if we don't? In the latter case we should remove these options because then we always have the most up to date defaults.

Reply:
> Where's that config from? Perhaps add comment.

Fixed in `fixup! module: explain where defaultConfig comes from`

> Do we need to state all these options, or does joinmarket pick the defaults if we don't?

AFAICT from https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/jmclient/jmclient/configure.py, we do need to state all these options.

Reply:
Ok